lib/transition.js

```diff
@@ -294,6 +308,24 @@ class Transition {
       }
       return undefined
     }
+
+    /**
+     * Check if transition prop is inside vendor specific rule
+     */
+    inVendorSpec (decl) {
+      let { parent } = decl
+      let { selector, type } = parent
+
+      if (decl._autoprefixerInVendorSpecific !== undefined) {
```

Comment:
This cache is not really necessary, since it content is pretty simple to calculate

Reply:
Removed it.